onnx/backend/test/case/node/cast.py

```diff
@@ -50,10 +52,25 @@ def export() -> None:
             ("FLOAT8E5M2", "FLOAT16"),
             ("FLOAT8E5M2FNUZ", "FLOAT"),
             ("FLOAT8E5M2FNUZ", "FLOAT16"),
+            ("FLOAT", "UINT4"),
```

Comment:
Should we add example from int4 to in8...?

Reply:
Sure, will do